packages/matter.js/src/fabric/Fabric.ts

```diff
@@ -267,6 +278,66 @@ export class Fabric {
         return Array.from(this.#scopedClusterData.get(cluster.id).keys());
     }
 
+    private getAsGroupSet(
+        operationalGroupSet: OperationalGroupKeySet,
+    ): TypeFromSchema<typeof GroupKeyManagement.TlvGroupKeySetStruct> {
```

Comment:
This seems like an identity operation?

Reply:
What do you mean?

Reply:
Ok, I changed it. I added a namespace function OperationalGroupKeySet.asTlvGroupSet ... i think so better? But correct, wrong as method in Fabric